libs/client/use-cases/part/list/shared/ui/src/lib/part-card/part-card-properties/part-card-properties.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="properties">
+  <div *ngFor="let property of partCardProperties" class="property">
+    <p class="tui-island__paragraph">
```

Comment:
We are not using the island component here. Why are we using these styles here ?